components/brave_wallet_ui/utils/account-utils.ts

```diff
@@ -174,7 +180,10 @@ export const keyringIdForNewAccount = (
   }
 
   if (coin === BraveWallet.CoinType.ZEC) {
-    if (chainId === BraveWallet.Z_CASH_MAINNET) {
+    if (
+      chainId === BraveWallet.Z_CASH_MAINNET ||
+      chainId === BraveWallet.LOCALHOST_CHAIN_ID
+    ) {
       return BraveWallet.KeyringId.kZCashMainnet
```

Comment:
There is no localhost for BTC (and ZEC afaik)

Reply:
Removed localhost checks for BTC and ZEC